packages/vue/src/generators/component/schema.d.ts

```diff
@@ -8,7 +8,6 @@ export interface Schema {
   fileName?: string;
   inSourceTests?: boolean;
   skipFormat?: boolean;
-  unitTestRunner?: 'jest' | 'vitest' | 'none';
```

Comment:
Shouldn't we still have `none`?

Reply:
No, component generator should never have this in the first place.